fe2-android/app/src/main/java/com/github/dedis/popstellar/utility/error/DataHandlingException.kt

```diff
@@ -0,0 +1,30 @@
+package com.github.dedis.popstellar.utility.error
+
+import com.github.dedis.popstellar.R
+import com.github.dedis.popstellar.model.network.method.message.data.Data
+
+open class DataHandlingException : GenericException {
+  @Transient val data: Data
+
+  constructor(data: Data, message: String?) : super(message) {
+    this.data = data
+  }
+
+  constructor(data: Data, message: String?, cause: Throwable?) : super(message, cause) {
+    this.data = data
+  }
+
+  override val message: String?
+    get() =
+        """
+               Error while handling data : ${super.message}
+               data=$data
+               """
+            .trimIndent()
```

Comment:
alignment seems off again, you might want to check where `"""` shows up in the codebase.

Reply:
Yeah, I just fixed every string that was replaced with """